### PR TITLE
add optional alert config

### PR DESCRIPTION
**Pull request by imjasonh**
Remaining stuff I'd like to make configurable:
- alert thresholds
- auto-close timeout

### CODE REVIEW

File: alert.tf [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2023 Chainguard, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Create an alert policy based on the uptime check.
+resource "google_monitoring_alert_policy" "uptime_alert" {
```
Comment (mattmoor) [inline alert.tf:7]: I think we need a way of plumbing notification channels into here for this to be useful.

Comment (imjasonh) [reply alert.tf:7]: Yeah that'll happen in the next PR.

Comment (imjasonh) [reply alert.tf:7]: Done, that was easy.

**Review by mattmoor — APPROVED** [review]
This LGTM, but let's release this carefully after the holiday

**Comment by imjasonh** [comment]
Sounds good. I'm merging this to unblock demoing some integrations with terraform-cloudrun-glue dashboards, but won't release anything.

**Pull request merged**